docs/src/main/asciidoc/amazon-sns.adoc

```diff
@@ -0,0 +1,445 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+= Quarkus - Amazon SNS Client
```

Comment:
Please no new line between the comment and the title. I don't know why but the website generator is very sensitive to that...

Reply:
fixed also dynamo & s3 guides as they had same issue